search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -46,21 +49,29 @@ function setCountryTrends(countryCode) {
             let articlesId = 'trend' + i + 'Article';
             let articlesHtml = 'Search results: <br>';
```

Comment:
Do we need to say search results? I think it is pretty obvious what it is.

Reply:
Not sure about this. @lawrencezjh @hayleejane3 @ntarn

Reply:
It is quite obvious, but depends on whether the user otherwise has enough indication on what theyre getting.